src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -917,15 +1137,18 @@ private void RebuildRegionConnections()
                 continue;
 
             // Check if connections should be highlighted
-            // (Unknown patches should not highlight connections)
```

Comment:
What's the reason behind this change, making unknown patches now be able to highlight?

Reply:
I think that it now makes more sense since it is connected anyway and user can click on it

Reply:
![image](https://github.com/user-attachments/assets/32fbafd6-213b-4f42-9c30-8dbfaccda731)
Like, in my opinion if I click this patch then the connections should be highlighted even if they fade away

Reply:
Yeah, that makes sense to me (the code was probably originally made for things to not look weird when something was partially visible). BTW that screenshot shows some aliasing artifacts, is the antialias option on for the lines already or not? If it isn't on already it might help a bit with the visuals.